### PR TITLE
Create Form, Input Labels for name, topic, and Level, add submit Butt…

### PR DESCRIPTION
**Pull request by abieber1408**
…ons, add Button to link with QuestionCard and link to it, add reset Button, styling, create screen for video

### CODE REVIEW

File: components/ContinueButton/index.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  background-color: #35268c;
+  border-radius: 15px;
+  color: #efedfa;
+  font-size: 15px;
+  cursor: alias ;
+  float: right;
+  width: 35%;
+  margin-botton: 30vh;
+  margin-right: -4vw;
+  margin-top:-8vh;
```
Comment (AnjaSiewert) [inline components/ContinueButton/index.js]: To make your code shorter, you could put all values in the prop margin, like this: margin: -8vh -4vw 30vh 0;

File: components/LevelForm/index.js [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+export const StyledLevelLabel = styled.label`
+  display: flex;
+  flex-direction: row-reverse;
+  font-weight: bold;
+  height: 30px;
+  width: 110%;
+  borde-radius: 100px;
+  margin-bottom: 0px;
+  margin-left: -4vw;
+  margin-top: -6.5vh;
```
Comment (AnjaSiewert) [inline components/LevelForm/index.js]: same here, you could sum up these in margin

File: components/LevelForm/index.js [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+  );
+}
+
+export const StyledLevelLabel = styled.label`
+  display: flex;
+  flex-direction: row-reverse;
+  font-weight: bold;
+  height: 30px;
+  width: 110%;
+  borde-radius: 100px;
```
Comment (AnjaSiewert) [inline components/LevelForm/index.js]: spelling mistake: it should be border-radius

File: components/LevelForm/index.js [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+    <StyledLevelLabel onSubmit={handleLevelSubmit} htmlFor="select-level">
+
+      <select id="select-level" name="select-level" onChange={(event) =>   onLevelChange(event)} >
+          <option value={0}>Level:</option>
+          <option value="1">1</option>
+          <option value="2">2</option>
+          <option value="3">3</option>
+          <option value="4">4</option>
+        value={selectedLevel}
+        onChange={(event) => setSelectedLevel(event.target.value)}
```
Comment (AnjaSiewert) [inline components/LevelForm/index.js:23]: looks a bit odd, seems that prettier is not working

File: components/QuestionCard/index.js [hunk 5]
```diff
@@ -50,65 +47,101 @@ export function ButtonGrid() {
...
+      <Heading/>
+      <StyledImage src={imageSrc}  width={135} height={88}/>
+      <p>{topic}</p>
       <ResultContainer>
         Level
-        <div>1 </div>
+        <div>{level}</div>
         Score
         <div>{score()} / {questionIds.length}</div>  
         <h1 style={{ position: "relative" }}><Image
```
Comment (AnjaSiewert) [inline components/QuestionCard/index.js]: I think inline styling is not the best solution, you should use styledComponents or global styling

File: components/QuestionCard/index.js [hunk 6]
```diff
@@ -50,65 +47,101 @@ export function ButtonGrid() {
...
           src={"/images/star.png"}
           alt="star"
           width={45}
           height={45} />
-        <dl style={{ position: "absolute", top:-19, left: 19}}>
-       <h2>{newStarScore}</h2>
+          <dl style={{ position: "absolute", top:-15, left: 19}}>
```
Comment (AnjaSiewert) [inline components/QuestionCard/index.js]: same here: you should use styledComponents or global styling

File: components/TopicForm/index.js [hunk 7]
```diff
@@ -0,0 +1,61 @@
...
+export const StyledFormLabel = styled.label`
+  display:flex;
+  flex-direction: row;
+  font-weight: bold;
+  height: 40px;
+  width: 100%;
+  borde-radius: 100px;
+  margin-bottom: 5px;
+  margin-left: 6vw;
+  margin-top:  -11vh;
```
Comment (AnjaSiewert) [inline components/TopicForm/index.js]: same here, border-radius spelling mistake and margin could be summed up in one prop.

**Review by AnjaSiewert — COMMENTED** [review]
LGTM in general, just some small suggestions from my side, see my comments.

One thing I noticed in general: You created a few styledComponents, e.g. a few button components. To make your code shorter and to not repeat yourself you can create one styledComponent called StyledButton and work with props in the respective buttons. This would save a lot of work for you as well :)

File: components/ContinueButton/index.js (hunk 1 above)
Comment (abieber1408) [reply components/ContinueButton/index.js]: Thanks!

File: components/LevelForm/index.js (hunk 2 above)
Comment (abieber1408) [reply components/LevelForm/index.js]: Thank You!.

**Review by alinawinter — APPROVED** [review]
Like we just spoke about, you do alot of absolute positioning with px in your app, instead of using flexbox or grid, which makes the app less responsive, but as this will be a bigger project for the entire app to adapt, maybe you can do this in another PR. Other than that lgtm :-)

File: components/QuestionCard/index.js [hunk 8]
```diff
@@ -0,0 +1,189 @@
...
+  div.score {
+    display: flex;
+    width: 33%;
+    height: 100%;
+    align-items: flex-end;
+    justify-content: space-around;
+  }
+
+`;
+export const StyledImage = styled.div`
```
Comment (PReichetanz) [inline components/QuestionCard/index.js:172]: Just to ensure what this styled component is doing (and it's not an image):
```suggestion
export const StyledImageContainer = styled.div`
```

File: components/QuestionCard/index.js [hunk 9]
```diff
@@ -0,0 +1,189 @@
...
+  display: flex;
+  width: 33%;
+  padding: 30px;
+`;
+
+const Section = styled.div`
```
Comment (PReichetanz) [inline components/QuestionCard/index.js:178]:
```suggestion
const Section = styled.section`
```

File: components/StartForm/index.js [hunk 10]
```diff
@@ -0,0 +1,60 @@
...
+  display: grid;
+  grid-template-columns: 1fr;
+  grid-template-rows: auto 1fr auto;
+  grid-template-areas: 
+    'header'
+    'main'
+    'footer';
+`;
+
+const Video = styled.div`
```
Comment (PReichetanz) [inline components/StartForm/index.js:56]:
```suggestion
const VideoContainer = styled.div`
```

**Review by PReichetanz — APPROVED** [review]
Good work! I've left only some naming ideas, but feel free to merge the PR anyways!

File: components/QuestionCard/index.js (hunk 8 above)
Comment (abieber1408) [reply components/QuestionCard/index.js:172]: Thank you Paul. Yes this is right. A very hard Work over the weekend. I will make the changes before merging.

**Pull request merged**